packages/opub-ui/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { forwardRef } from 'react';
+import styles from './Badge.module.scss';
+import cx from 'classnames';
+import { BadgeProps } from '@ui/types/badge';
+import { Complete, Incomplete, Partial } from './icons';
+
+const progressIcons = {
```

Comment:
Instead of creating these svgs, try to implement how they are implementing using css only 
https://github.com/Shopify/polaris/tree/main/polaris-react/src/components/Badge